*** innobackupex-1.5.1.orig	2013-11-27 21:04:46.000000000 +0900
--- innobackupex-1.5.1	2013-11-27 21:04:29.000000000 +0900
***************
*** 3976,3981 ****
--- 3976,3988 ----
              }
          }
      }
+ 
+     # backup .mrn files
+     rewinddir(DIR);
+     my @mrn_file = grep(/\.mrn(?:\..+)?/, readdir(DIR));
+     foreach $file (@mrn_file) {
+         backup_file("$source_dir", "$file", "$backup_dir/$file")
+     }
      closedir(DIR);
  
      if ($option_rsync) {
